src/translations/en.json

```diff
@@ -2810,20 +2810,20 @@
             "caption": "Network"
           },
           "groups": {
-            "add_group": "Add Group",
+            "add_group": "Create Group",
```

Comment:
Switch to Add is other places but here we have create?

Reply:
Remove and Delete are defined quite similarly, but the main difference between them is that delete means erase, while remove denotes take away and set aside but kept in existence. In most cases, Create can be paired with Delete, and Add can be paired with Remove. 

In case of groups, this doesn't exist yet and is created.